_posts/2022-06-05-May.md

```diff
@@ -0,0 +1,80 @@
+author: MisterE
+contributors: ["MisterE", "Rubenwardy"]
+description: >
+  This month, Minetest 5.5.1 is released, nodebox rendering gets a speedboost, we get a new mapping mod, and Minetest's first walkable vehicle. Exile and Nodecore get major improvements as well. 
+image: /static/blog/2022_May/izzyscastle.webp
+forum_topic: https://forum.minetest.net/viewtopic.php?f=18&t=28105
+tags:
+  - this_month
+---
+	First, we should celebrate a major milestone. For the first time in a long time, Minetest finally has fewer than 1000 open issues. A big thank you to the Development Team for speeding up the pace of development, and to Zughy for triaging the issues and closing the ones that could safely be closed.
+	
+  This month, Minetest 5.5.1 is released, nodebox rendering gets a speedboost, we get a new mapping mod and minetest's first walkable vehicle. Exile and Nodecore get major improvements as well. 
+
+
+
+*  [Engine](#engine)
+*  [Games](#games)
+*  [Mods](#mods)
+*  [Art/Builds](#art)
+*  [Server News](#s-news)
+*  [In Other News](#o-news)
+
+
+## Engine News <a name="engine"></a>
+	Minetest 5.5.1 was released this month. It fixes a whole list of bugs ( see [here](https://dev.minetest.net/Changelog#5.5.0_.E2.86.92_5.5.1) ). x2048 worked on improving rendering. Now, snow and other simple nodeboxes do not draw hidden faces. Of course, this only applies to opaque nodes.  Also, x2048 improved soft shadows.
+## Game News <a name="games"></a>
+
+### Nodecore
+	Since March, Warr1024 has been working on translations for Nodecore. Now there are practically complete translations in Russian, French, and German.
+	Also, many things have been improved with the optics automation systems. The new features especially improve optics' detection capabilities.
+	
+### Exile
+	Exile 0.3.7 is out! Among other improvements, the highlights are: improved airboats, multiplayer node protection for some valuable items, and the beginnings of translation support.
+	
+### Cascade
+
+{% include figure.html src="/static/blog/2022_May/cascade.png" caption="A Cascade of mazes" %}
+
+	Cascade is a new maze game. While other maze games have been released, this one is unique in that each new maze level is laid out before you before you enter it, giving you a chance to plan your route to the exit. Once entered, the walls are too high to see over. It is an interesting take on the maze game theme.
+	
+## Mod News <a name="mods"></a>
+
+### Xmaps
+{% include figure.html src="/static/blog/2022_May/xmaps.png" caption="Treasure Maps for Minetest" %}
+
+[xmaps](https://content.minetest.net/packages/erlehmann/xmaps/) is a new mod by [erle](https://content.minetest.net/users/erlehmann/) that adds treasure-maps to the HUD. You get them by rightclicking a mapping kit. You can mark the spot with an X by rightclicking a node. Treasure Hunt, anyone?
+
+### Steampunk Blimp
+{% include figure.html src="/static/blog/2022_May/blimp_1.jpg" caption="Exploring the Skies" %}
```

Comment:
Is there a screenshot without the chat / hud?

Reply:
we can cut the screenshot...